adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java

```diff
@@ -673,14 +673,16 @@ public void testGetAuthTokenInBackgroundVerifyErrorMessageNotSupported() throws
         }
     }
 
-    public void testGetAuthTokenInBackgroundVerifyErrorNoNetworkConnection() throws NameNotFoundException {
+    public void testGetAuthTokenInBackgroundVerifyErrorNoNetworkConnection() throws NameNotFoundException,
+    OperationCanceledException, IOException, AuthenticatorException {
         final String acctName = "testAcct123";
         final AuthenticationRequest authRequest = createAuthenticationRequest("https://login.windows.net/omercantest", "resource", "client",
                 "redirect", acctName.toLowerCase(Locale.US), PromptBehavior.Auto, "", UUID.randomUUID(), false);
         final FileMockContext context = new FileMockContext(getContext());
         context.setConnectionAvaliable(false);
         final AccountManager mockedAccountManager = getMockedAccountManager(AuthenticationConstants.Broker.BROKER_ACCOUNT_TYPE,
                 AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME);
+        setMockProxyForErrorCheck(mockedAccountManager, acctName, AccountManager.ERROR_CODE_NETWORK_ERROR, ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription());
```

Comment:
can you also add a e.getMessage()!= null in the place where the crash was as well.